sample-lnd.conf

```diff
@@ -1118,6 +1118,12 @@ litecoin.node=ltcd
 ; If non zero, LND will use this as peer port for the embedded etcd instance.
 ; db.etcd.embedded_peer_port=1235
 
+[postgres]
+; Postgres connection string
+; db.postgres.dsn=postgres://lnd:lnd@localhost:45432/lnd?sslmode=disable
+
+; Postgres connection timeout. Leave unset for no timeout.
```

Comment:
Since this is a `time.Duration` we should add the ` Valid time units are {s, m, h}.` comment.